x/bucket/service/ipfs_generate_bucket.go

```diff
@@ -21,11 +21,11 @@ func GenerateBucket(sh *shell.Shell, b *bt.BucketConfig, address string) (*rt.Se
 		return nil, fmt.Errorf("failed to Get Directory %s CID on IPFS Node, %e", path, err)
 	}
 	service := b.GetDidService(address, res.Hash)
-	/*
-		err = sh.Publish(res.Hash, path)
-		if err != nil {
-			return service, fmt.Errorf("failed to Publish Directory %s CID %s to IPNS in IPFS Node, %e", path, res.Hash, err)
-		}
-	*/
+
+	err = sh.Publish(res.Hash, path)
```

Comment:
isn't this being done within motor? do we need to publish here?

Reply:
This function is only called inside motor. It was removed from AllocateBucket